javascripts/controllers/DetailCtrl.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+
+app.controller("DetailCtrl", function ($routeParams, $scope, ContactService){
+
+    $scope.contact = {};
+
+    console.log("ContactId", $routeParams.id);
+
+    const getContact = () => {
+        ContactService.getSingleContact($routeParams.id).then((results) => {
+            console.log("results.data", results.data);
```

Comment:
Make sure to remove console logs from your PR.

Reply:
@morecallan @zoeames @rousell  GTG!!!